### DISCUSSION TITLE
for the Galaxy Brain badge

### DISCUSSION THREAD

**Discussion by AngJianming** [recodehive/Stackoverflow-Analysis#511] [Q&A] [answered]
for da badge

**Comment by AngJianming** [comment]
here

**Reply by Advanced-hub** [reply]
Bro I think you still cannot get the badge la

The description says:

You made the decision to disable the ability to earn Achievements in our Community in order to **discourage users from participating in coordinated or inauthentic activity** like **rapid questions and answers** in order to earn badges. You can learn more about this decision in our announcement post here Achievements will no longer be available in the Community.

Note that GitHub's Acceptable Use Policies prohibits coordinated or inauthentic activity like rapid questions and answers. If users create discussions of this nature, it may result in a temporary or indefinite block from the Community.

This change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

Thanks for understanding!

**Reply by AngJianming** [reply]
eh yea le damnn then you want to help me mou?

**Reply by Advanced-hub** [reply]
Yea can, you help me also then

**Reply by AngJianming** [reply]
yea can ofc

**Reply by AngJianming** [reply]
I create new discussion ba

**Comment by Pranjal-88** [comment]
here

**Comment by abubakaristiak** [comment]
The Galaxy Brain badge is part of GitHub's achievements system, introduced to recognize users' contributions and engagement within the platform. This badge is awarded to users who provide accepted answers in GitHub Discussions, highlighting their expertise and helpfulness in the community.

The badge has multiple tiers, each representing a higher level of contribution:

Default (x1): 2 accepted answers
Bronze (x2): 8 accepted answers
Silver (x3): 16 accepted answers
Gold (x4): 32 accepted answers

**Reply by AngJianming** [reply]
Yes that's right haha, I just wanted to get the Default version looks supa cool

**Reply by sanjay-kv** [reply]
@AngJianming https://stream.recodehive.com/github

i made a playlist for this. if any doubt

**Comment by abubakaristiak** [comment]
for the badge...

**Comment by sandipanrakshit34** [comment]
for the badge please mark me.....

**Reply by Pranjal-88** [reply]
Open a discussion

**Reply by abubakaristiak** [reply]
for the badge please mark me.....

**Reply by AngJianming** [reply]
I can't mark it, if you want to get the badge you need to open a new discussion in Q&A

**Reply by AngJianming** [reply]
Then someone needs to comment on the discussion, only then you can mark their comment as Answer, then you'll get the galaxy brain badge

**Reply by AngJianming** [reply]
remember to repeat the process twice

**Reply by abubakaristiak** [reply]
Thanks brother...❤️

**Comment by abubakaristiak** [comment]
for the badge please mark me.....

**Comment by OrasanuAna** [comment, marked as answer]
Please mark me for the badge. Thank you! :)

**Reply by Pranjal-88** [reply]
Hey ! Open a discussion !!

**Comment by ghost** [comment]


**Reply by sandipanrakshit34** [reply]
sure

**Comment by muhammadyaqoobmuet** [comment]
The Galaxy Brain badge is part of GitHub's achievements system, introduced to recognize users' contributions and engagement within the platform. This badge is awarded to users who provide accepted answers in GitHub Discussions, highlighting their expertise and helpfulness in the community

**Comment by 955n** [comment]
can i get bro

**Comment by MrH1deZ** [comment]
Can you guys also help me for the badge?

**Comment by TMBilalTM** [comment]
Can you guys also help me for the badge?

**Comment by sandipanrakshit34** [comment]
hey please give me a star on this repository and help me to achieve the badge:

https://github.com/sandipanrakshit34/sandipanrakshit34

**Comment by PedroHenriques1** [comment]
you can help me also?

**Comment by TMBilalTM** [comment]
Please...

**Comment by TMBilalTM** [comment]
Please...

**Reply by im-ahmed-hasan** [reply]
@TMBilalTM Please star my repo and I’ll star yours -> https://github.com/im-ahmed-hasan/selenium-cdp-automation

**Reply by TMBilalTM** [reply]
I starred, bro

**Reply by im-ahmed-hasan** [reply]
@TMBilalTM which repo would you like me to star?

**Reply by TMBilalTM** [reply]
passwordgame bro pls.

**Comment by ChadenBA** [comment]
here ...

**Comment by ChadenBA** [comment]
Please mark me for the badge. Thank you! :)

**Reply by chadenbenammar** [reply]
testtt

**Reply by im-ahmed-hasan** [reply]
@chadenbenammar @roseewood Please star my repo and I’ll star yours -> https://github.com/im-ahmed-hasan/selenium-cdp-automation

**Reply by im-ahmed-hasan** [reply]
@roseewood which repo would you like me to star?

**Reply by im-ahmed-hasan** [reply]
Done @roseewood

**Comment by TMBilalTM** [comment]
Please mark me for the badge. Thank you! :)
